cli/src/main/scala/org/apache/celeborn/cli/master/MasterOptions.scala

```diff
@@ -45,7 +45,8 @@ final class MasterOptions {
 
   @Option(
     names = Array("--send-worker-event"),
-    paramLabel = "IMMEDIATELY | DECOMMISSION | DECOMMISSION_THEN_IDLE | GRACEFUL",
+    paramLabel =
+      "IMMEDIATELY | DECOMMISSION | DECOMMISSION_THEN_IDLE | GRACEFUL | RECOMMISSION | NONE",
```

Comment:
should we just ` EventTypeEnum.values().toStream.map(_.name()).mkString("|")` here as well?

Reply:
@akpatnam25, when using `EventTypeEnum.values().toStream.map(_.name()).mkString("|")`, there is complication error that is `MasterOptions.scala:50: annotation argument needs to be a constant; found: scala.Predef.refArrayOps[org.apache.celeborn.rest.v1.model.SendWorkerEventRequest.EventTypeEnum](org.apache.celeborn.rest.v1.model.SendWorkerEventRequest.EventTypeEnum.values()).toStream.map[String, scala.collection.immutable.Stream[String]](((x$1: org.apache.celeborn.rest.v1.model.SendWorkerEventRequest.EventTypeEnum) => x$1.name()))(immutable.this.Stream.canBuildFrom[String]).mkString("|")`.

Reply:
that is interesting. i am fine with your original change then.